### ISSUE TITLE
Improve paragraph accessibility in limits reached modals

### ISSUE THREAD

**Issue by gtsiolis** [gitpod-io/gitpod#7161]
### Problem to solve

Text accessibility in limits reached modals fails to pass WCAG AA for normal text which is the WCAG 2.0 level we're aiming for.

https://github.com/gitpod-io/gitpod/blob/c715b3521f2a2d68937cdd44ecfa35674b3c10c5/components/dashboard/src/start/CreateWorkspace.tsx#L224

https://github.com/gitpod-io/gitpod/blob/c715b3521f2a2d68937cdd44ecfa35674b3c10c5/components/dashboard/src/start/CreateWorkspace.tsx#L230

While the current `text-base` colors for the dark theme in most case passes WCAG AA for normal text when used with the default dark background, it fails when used inside a modal which is using a different background color (`dark:bg-gray-900`).

https://github.com/gitpod-io/gitpod/blob/0e0396942d050a35b469bee5615d0b0177894a89/components/dashboard/src/components/Modal.tsx#L53

### Proposal

Change the paragraph text color inside modals for the dark theme. 🌔 

| BEFORE | AFTER |
|-|-|
| <img width="566" alt="modal-before" src="https://user-images.githubusercontent.com/120486/145484744-6d08019d-aac3-494f-9fe3-9f8d238231e1.png"> | <img width="566" alt="modal-after" src="https://user-images.githubusercontent.com/120486/145484746-8d483514-6080-40a1-8572-76bb9c776b9d.png"> |

The following fix could suffice:

```diff
diff --git a/components/dashboard/src/start/CreateWorkspace.tsx b/components/dashboard/src/start/CreateWorkspace.tsx
index b2274168..7b870ba5 100644
--- a/components/dashboard/src/start/CreateWorkspace.tsx
+++ b/components/dashboard/src/start/CreateWorkspace.tsx
@@ -221,13 +221,13 @@ function LimitReachedModal(p: { children: React.ReactNode }) {
 
 function LimitReachedParallelWorkspacesModal() {
   return <LimitReachedModal>
-    <p className="mt-1 mb-2 text-base">You have reached the limit of parallel running workspaces for your account. Please, upgrade or stop one of the running workspaces.</p>
+    <p className="mt-1 mb-2 text-base dark:text-gray-400">You have reached the limit of parallel running workspaces for your account. Please, upgrade or stop one of the running workspaces.</p>
   </LimitReachedModal>;
 }
 
 function LimitReachedOutOfHours() {
   return <LimitReachedModal>
-    <p className="mt-1 mb-2 text-base">You have reached the limit of monthly workspace hours for your account. Please upgrade to get more hours for your workspaces.</p>
+    <p className="mt-1 mb-2 text-base dark:text-gray-400">You have reached the limit of monthly workspace hours for your account. Please upgrade to get more hours for your workspaces.</p>
   </LimitReachedModal>;
 }
 
```

**Comment by championshuttler** [comment]
ping
